Doc/library/locale.rst

```diff
@@ -331,6 +331,8 @@ The :mod:`locale` module defines the following exception and functions:
       The function now always returns ``UTF-8`` on Android or if the UTF-8 mode
       is enabled.
 
+   On VxWorks, always return ``'UTF-8'``, the locale and the *do_setlocale*
+   argument are ignored.
```

Comment:
```suggestion
   argument is ignored.
```

Reply:
> 

"are" is used for "the locale" and "the *do_setlocale* argument".

This line is mostly the same as comment for Android at line 328:

On Android or in the UTF-8 mode (:option:`-X` ``utf8`` option), always
return ``'UTF-8'``, the locale and the *do_setlocale* argument are ignored.